### PR TITLE
Add special case for sendHttpRequest activity.

### PR DESCRIPTION
**Pull request by swuecho**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/elsa-workflows/elsa-website?pullRequest=34) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/elsa-workflows/elsa-website?pullRequest=34) before we can accept your contribution.<br/><br/>:x: swuecho [sign now](https://cla.dotnetfoundation.org/elsa-workflows/elsa-website?pullRequest=34)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/elsa-workflows/elsa-website?pullRequest=34) it.</sub>

File: docs/expressions/expressions-javascript.md [hunk 1]
```diff
@@ -85,6 +85,17 @@ Then you can access the `"Title"` field like this:
 
 > If your activity is a direct child of an HTTP Endpoint activity, you can access its output directly via the `input` variable, which will be an instance of `HttpRequestModel`.
 
+#### SendHttpRequest Activity
+
+The output property of the **SendHttpRequest** has two output properties:
```
Comment (sfmskywalker) [inline docs/expressions/expressions-javascript.md:90]: Please change this sentence to:

`The **SendHttpRequest** activity has two output properties:`

File: docs/expressions/expressions-javascript.md [hunk 2]
```diff
@@ -85,6 +85,17 @@ Then you can access the `"Title"` field like this:
...
+
+```csharp
+        [ActivityOutput] public HttpResponseModel? Response { get; set; }
+        [ActivityOutput] public object? ResponseContent { get; set; 
+```
+ 
+To access an **SendHttpRequest** activity with name `SampleRequest1`'s response content, use `activities.SampleRequest1.ResponseContent()`.
```
Comment (sfmskywalker) [inline docs/expressions/expressions-javascript.md:97]: Please change `an` to `a` so that it reads: `To access a **SendHttpRequest**`.

**Review by sfmskywalker — CHANGES_REQUESTED** [review]
Just two small change requests.
Thanks for the PR!

**Review by sfmskywalker — APPROVED** [review]

**Pull request merged**